---
title: "Final Project - The Platinum Age of Virus Discovery"
output:
  html_document:
    fig_caption: true
---

## Preamble
Please mark the following boxes with an X as appropriate:
```
- [ ] Completed the Request for Evaluation by Artem
- [ ] Answers assisted with generative AI (please indicate where)
```

# FINAL PROJECT ================================================================
# ==============================================================================
```
..==                                                                        ==..
..==           ██▒   █▓ ██▓ ██▀███   █    ██   ██████   ▓██   ██▒           ==..
..==          ▓██░   █▒▓██▒▓██   ██▒ ██  ▓██▒▒██    ▒   ▒▒ █ █ ▒░           ==..
..==           ▓██  █▒░▒██▒▓██  ▄█ ▒▓██  ▒██░░ ▓██▄ ░░  ░░  █   ░           ==..
..==            ▒██ █░░░██░▒██▀▀█▄  ▓▓█  ░██░  ▒   ██▒  ░  █ █ ▒            ==..
..==             ▒▀█░  ░██░░██▓ ▒██▒▒▒█████▓ ▒██████▒▒  ▒██▒ ▒██▒           ==..
..==             ░ ▐░  ░▓  ░ ▒▓ ░▒▓░░▒▓▒ ▒ ▒ ▒ ▒▓▒ ▒ ░  ▒▒  ░ ░▓░           ==..
..==             ░ ░░   ▒ ░  ░▒ ░ ▒░░░▒░ ░ ░ ░ ░▒  ░ ░  ░   ░▒ ░            ==..
..==               ░░   ▒ ░  ░░   ░  ░░░ ░ ░ ░  ░  ░ ░░ ░    ░              ==..
..==          ▓█████▄   ██████  ▄████▄ ██▒ ░ █▓ ██▀███ ▓██  ░██▓            ==..
..==          ▒██▀ ██▌▒██    ▒ ▒██▀ ▀█ ██░   █▒▓██ ▒ ██▒▒██  ██▒            ==..
..==          ░██   █▌░ ▓██▄   ▒▓█    ▄ ██  █▒░▓██ ░▄█ ▒ ▒██ ██░            ==..
..==          ░▓█▄   ▌  ▒   ██▒▒▓▓▄ ▄██▒▒██ █░░▒██▀▀█▄   ░ ▐██▓░            ==..
..==          ░▒████▓ ▒██████▒▒▒ ▓███▀ ░ ▒▀█░  ░██▓ ▒██▒ ░ ██▒▓░            ==..
..==           ▒▒▓  ▒ ▒ ▒▓▒ ▒ ░░ ░▒ ▒  ░ ░ ▐░  ░ ▒▓ ░▒▓░  ██▒▒▒             ==,.
..==           ░ ▒  ▒ ░ ░▒  ░ ░  ░  ▒    ░ ░░    ░▒ ░ ▒░▓██ ░▒░             ==..
..==           ░ ░  ░ ░  ░  ░  ░       ░   ░░    ░░   ░ ▒ ▒ ░░              ==..
..==             ░          ░  ░ ░          ░     ░     ░ ░  ░              ==..
..==           ░               ░           ░            ░ ░                 ==..
..==             ░ The Platinum Age of Virus░Discovery  ░                   ==..

At the dawn of the Molecular Genetics, Max Delbruck led a loose collection of
geneticists known as the "Phage Group". They were amongst the first to pick up
the reciever when the Molecular Revolution was calling. They had two unifying
philosophies:

(1) Hard Physics-style Scientific Inquiry
(2) They studied viruses, because that's the simplest model organism to do
    cool science in.
    
Computational Biology is having that kind of moment today. And the way to 
realize the Computational Revolution of biology is by choosing a strong model;
Viruses and Virus-like Agents!

Guidance from the Elders:

https://www.science.org/doi/10.1126/science.146.3642.347

..==                                                                        ==..
..==                                                                        ==..
..==                                                                        ==..
```

## Learner Objectives
```
By the end of this assignment, you will have taken the plunge into a real
research problem, there is risk and no certainty of "success", but as the old
saying goes, "If you can't fatally kill your hypothesis, then it was never alive
to begin with." Risk begets Reward.

It is up to you to find and guide the direction of the project, these questions
only give you a generic scaffold. As such, (and with the bonus marks) it will
be possible to do a technically good job and you'll do fine on the assignment.

A subset of you will rise to the occasion and knock it out of the park, that 
will reflect the 99% grades. If you earn 99% on this final, consider it a formal
job offer to join the RNAlab for the summer.

If you do an exceptional on this project and uncover something worth publishing,
I will encourage you to pursue it and go through with a publication.

Remember, you're a scientist if you choose to be.
```

## Assignment Objectives
```
For this project you're going to compile all the components for a short
"Genome Announcement" style scientific report, describing the RNA virome
of a species which you have been randomized. You do not need to re-write the
answers into a seperate paper, this assignment serves as the paper/methods 
template. You will know by the end if it's worth formalizing. In addition you
are to prepare a 6 minute video  presentation summarizing your work.

The goal here is *Data Exploration*, you are not expected to have every analysis
work and be super insightful, instead you will be graded on rationale, creativity
technical execution, and presentation of your analysis. Outcome does not affect
grade. 

Most analyses will be difficult or impossible to interpret, the fun part of
science  is we can't always know the difference. If you are able to
justify (hypothesis) the reason you performed a given analysis, and it's
inconclusive, you will recieve full marks for trying. You will be deducted (few)
marks if you include an analysis and it's not justified, or if you misinterpret
the analysis results. The _most common_ error is not justifying your conclusions
with reported data, or over-claiming conclusions. Be precise, be specific, and 
back up your statements with data! This is the feedback/learning mechanism of
peer review, it will guide you in the future.

Please include in the work when you judge you can make a *low*, *moderate* or
*high*  confidence conclusion. Also state if your results are inconclusive.
```


## Logistics
```
Inclusive in this document should be all supporting code, scripts, and/or 
software settings used to answer any questions in the assignment. If you use or
generate a data-object, such as .fasta, .pdb, .sto, .csv, etc..., include this
in your submission repository.

For Written portions please use block text like this and you can include 
references simply as square brackets [Babaian. 1989], with a references section
at the end.
```

## Virus Assignment
```
You are randomly assigned a Virus. 

## `data/virusAssignments.csv` 
  Gives the assignment of your virus. Each virus has a unique identifier (sOTU)
  as well as some summary statistics about where this virus was "found" in the
  SRA. You're also given a link to the assembly file from the "index run" which
  should contain your virus sequence.

## `data/virusRunObservations.csv`
  This file  contains a list of all SRA "runs" in which these  viruses were 
  reliably detected. You can use the sOTU identifier to get an  estimate overview
  for your virus. The final column in this file contains the  barcode sequence
  detected in that particular run.

```

## Example "Virus Announcement" references
(Emulate these! Read for inspiration.)

- [Obelisks](https://www.sciencedirect.com/science/article/abs/pii/S0092867424010912)
- [Odysseus](https://academic.oup.com/ve/article/10/1/veae040/7668577)
- [Inocles](https://www.biorxiv.org/content/10.1101/2024.09.30.615745v1)
- [Borg](https://www.nature.com/articles/s41586-022-05256-1)
- [Zeta Elements](https://www.nature.com/articles/s41586-021-04332-2)
- [Ambivirus elements](https://www.nature.com/articles/s41467-023-38301-2)
- [Bornaviruses](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8321990/)
- [Pepino Mosiac Virus](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952647/)
- [Lancehead filovirus](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8498845/)
- [Perch filovirus](https://journals.asm.org/doi/10.1128/mra.00028-23)
- [Locarnaviruses](https://link.springer.com/article/10.1007/s00705-023-05908-1)


# ==============================================================================

## Question 1 : Framing the story `(10 points)` --------------------------------
**Q.** `(1 point)` Write a title for your paper which describes your key finding.

**A.** TODO

**Q.** `(bonus point)` Include a pun in your title which relates to a theme or motif
used throughout your paper.

**A.** TODO

**Q.** `(1 point)` Give your virus a latin binomial name, and explain the etiology
of the name and how it relates to the story of your virus.

**A.** TODO

**Q.** `(8 points)` For any given paper, 90% of the readership will only read your
abstract. How many people go on to read the rest of your paper will depend
on your ability to capture peoples interest in the abstract. (hint: write the
abstract last)

Read *How to Construct a Nature summary paragraph* and write a short abstract
closely following this structure. Max 300 words.

Link: https://www.nature.com/documents/nature-summary-paragraph.pdf

**A.** TODO


## Question 2 : Digital Ecology  `(20 points)` ---------------------------------

In brief, ecology is the relationship of an organism to biotic and abiotic
factors. Consider the sample source for this virus and describe the specifics
(e.g. run accessions, study authors, associated species, ...) and generics 
(e.g. type of ecosystem, country/continent, common associated host class like
Mammals).

All viruses occur in more than one dataset (SRA run), compare and contrast
the ecology of your index case, and the broader ecology of all virus-containing
datasets. What commonalities, if any, emerge?

Organize your answer in notes and ideas into at two or three topic sub-sections,
with a heading for each. This will form the "body" of your story. For
each sub-heading, develop a logical progression of notes or short sentences.

This should be methodology oriented, explain the details of what and how you
performed the analysis, linking logical ideas together. If you ran any program
with a non-standard parameter, be sure to include that information.

Guiding questions:
- Is there an associated publication with your index case?
- How many different "bioProjects" does your virus land in? Are there more
  publications describing those?
- Which dataset(s) is the virus found in?
- How does this observation compare to the "expected" niche of related viruses?
- What is the origin(s) of these dataset(s), technology and biology.
- Can you observe a common factor which links these datasets together and
  may explain the presence of the virus?
- How can you measure and quantify your observations. More importantly, how do 
  you translate this into a hypothesis?

```
# Example Answer
**Identification of a T. gondii-associated viral sequence**

Using SerratusSQL to screen for novel human neuroinflammatory RNA viruses, 
we searched the BioSample database [Barrett et al. 2012] for SRA libraries
annotated with metadata describing the sample as 
  (1) human and 
  (2) central nervous system.
  
SQL query:
`SELECT * FROM tismap_summary 
  WHERE biosample_tags LIKE ‘%neuron%’ 
  AND scientific_name ‘Homo sapiens’ 
  AND percent_identity 90 
  ORDER BY coverage desc;`

From the 7,675,502 sequencing runs, 483,173 (6.29 percent) runs were annotated
as `central nervous system` samples, of which 82,454 (17.07 per cent) were
annotated as human. 

We further filtered to ‘neuron’ annotated datasets containing an unknown virus
(<90 per cent RdRp amino acid identity, detected by palmscan, 
[Babaian and Edgar 2022]). This yielded a ten matching libraries.

By contig coverage, three of the highest-expression libraries (SRR1205923,
SRR1204654, and SRR1205190) originated from one BioProject (PRJNA241125), which
we manually inspected to identify a divergent RdRp fragment (palmprint:
u150420_halalDiploma).

We term this BioProject’s associated paper ‘the Ngô study’, wherein the authors
infected three human cell types (neuronal stem cells, neuronal differentiated
cells, and monocytes) in vitro, with nine different strains of the
apicomplexan parasite T. gondii or a mock control [Ngô et al. 2017].

We assembled all 117 available Ngô mRNAseq runs de novo and with palmscan
identified a 3,202 nucleotide (nt) putative viral contig with a coding-complete
989 amino acid RdRp (see below).

A blastp search on the translated protein revealed highly divergent RdRp
homologs; the highest identity match was an unclassified Riboviria RdRp
(search date: 18 June 2023, accession: QIM73983.1, percent-identity: 54.13 per
cent, query-coverage: 40 per cent, e-value: 9e-133).

>OZ005311.1 Apocryptovirus odysseus RUB 1 Segment 1. RNA-dependent RNA polymerase, complete CDS
CTTTTTTTACCGGTAATCGGTAAGCGCGCCTCAGGGTCCGCTTTGGTGTAAAACCAGCGATACCGGATCG
GTAGAATGTCTCTACCCGATATTGTTCCCGTGCATGACTGCAGAACAAGCGCGCAAGAAGTTCGCGATGG
CACTATTAAAGTTGACCATCGTTGGTACCTAGATTTCCTAGATACCAATCCCGGATCCATTTTACATGGA
TGGGACGAAATCCCGTCGTATGCAAGAAAGAGAAGAGTTATACGCTCTTTCCCTTTCGAGCTAAAGACGG
ACGTGATAGATAAACTATCACGTGCAGATTCCGAATTCGTTAAACGAATCGTGAATCTGGTTCTGGCGTT
TGAAGATAATTTACTTTTCTCAGACCCAGAACATTTCCCCAGAATATTTCGTTCTAGGGAATACAACCGA
TTGTTCAAATGGGTTGTATCCAGTGAGGTATACTCTCCTGGTTCTCCTGTAGATGAATGGAAATCATTCA
TCCAGGTCGTCCGATGGCATGCATTTCGCAGTCAGACGGAAAAGCCAGACGGGCCGCGGAATTTCCCCGG
ATTCCGTCGTCAGAAGGGGTTGGATGAACTTCACCACTTCTGGGCTGTTCTGGTACCCTGGTTAATCCCG
GTTTGGAGGGAAGGGTTAACCTCTAAACTCCAATGTACCAGATTAATGCATTTCCTAAGTTCTAGGAATA
TGCCTGCGGCACCCAAGAAGAAGCGTAACGCTTCCTTGGAAGAACACTTCCGGGTTTTAACCTCGGAAGG
GCCGGACCTGGCCAGTCGTAACGAGTGGCTCAGAGGAGCCGCCTTCCATTACGGAAGGCGACTGCGCTCA
GAGAGCGGAGAGACCATATCAGGTGCTCATCTCTCAGTGACCAATTCTGCGTCCTTCCTATCCTCTGTTA
AAGAGGGTGGTCGTGGCGTAGAAACGGCCAGCTTTTTCCGATCCTGGATCATGAAGAAACCAGAAACTAC
CGAACAAAGGTTAACTTGGTTCGGAAAGCCGTACTGGCTGGAAAAAGGAACTCCGCAATGGGAGACTATG
TGCCGTGAGGTTAAACCTCAATCGGGAGAACCTGGCGACTCTTCGTATTTAACGGAAGTCGACTTCACAC
CGGGTGCATTTCGGTATGAAGATCCCATATATTGTTTAGATCAGTATACGGGATACCAGATCTTCCAGTG
GTCTCTGGAAGAAGGGATTTCCCGCGGGTATATCCGGGGAAAACCCTATCGCGATCCTGATGGTTTATCC
ATTACAGGAAAGGCCCCATTAATTAGGGCTTCCGCGATTGGCGAACCTGGGAATAAATCCCGGGTCGTCA
CGGTTGGGGAGGGTTGGTTAACAATCTTCCTCCAGCCGTATGCCCACGCGCTTATAGGTATACTCAAGCG
CGATTCGGATGCCAAAGCCGGACTCTCAAGAGCCTGGCAAGGATTCGAATACGTTAAACGTTGGGCAGGT
AGGGGTAAACCTCCACCAGAGGAACAGAGGTTTATCCTCTCTTCCGATCTCAAAACCGCTACGGACTACT
GCCCGCATGGTTACTCTCGATCCTTGTTAAAAGGATTCATAGATGGGTTAGGACTGGAAACAGACCTAAC
CAATCTATGGGTTGATCTCCTCACCTGCGGTAGAACATATGTTTCACCGTTCGGTAAAAAGATCACCACC
CGAGGTATACTCATGGGTGACCCGGGAGCGAAGCTCGTGCTATCATTGTTTAACAAGGTAGCCGAGATGG
AGGCTCAAGTACGATTTAATATCGGTACTCACCTCCGAATGTCTGGGAAGTTCCTTCTCAGACATCTAGA
GACTGACGGCCTTCACGGTCCTCAGTCTCGATTATTTGCCTTTGCAGGAGATGACCATGTGGCTCTAGGC
CCAGTCAACTACCTTCGGCAAATCACAGCTTCACACATCCGGAACGGAATGTGTGTGTCGACTACCACGA
ATTATATTTCGTGTGTAGCAGCTGTGTACTGCGAGGAGATGGTTTTCATCCGCTCGACAGATGTGTGGAA
GTATTGGGGAAGTAACATTCCCCTATACAAATCTCCGTATCAGGAGAATCCACATGTAGACGCTCTTAAA
TTGCGACTACTTTCCCTGTGTCGTAAAGAACACGAAGGGAAAAACGATGCCAACCCTGCAATCGGTATGT
CTTCCTCGTTACGGGGAATGATACACTGGTTTGCAGAAGGGTGGGAAACCACCCGTGGCATCGTCAGCAA
GAGATTCCGGCAAAAAATGCGGGGGTTAATCCCTGAAAATCTCTTCCTTGCGGCACTCCCCAAGGAACTT
GGTGGAATAGAGTGCCCAGTCTTTGACGTTCCTGATACGGAGCTTCAAAGACACTTCCTCGAATTGCCAG
AGCTACACAAACGTGTAATAGCCCTGGTCCTGAGGAAGGATGAGAAAACAAACCCGTCGTTCCGACGGAT
GTTATCTCAGATACCTTCGATGAGCGCTCTACGCGGGGTTGACCCAGACTCAATCGAAGAATCCGTTCGG
GAGGTTTTATCCTCCGAACTGTGTCAGTCTTTACGACTGTCAGAAATCCGGGTTAAACTCGGATTTTCTG
AAGTTGCCTGGGAGCAGTTACGCTTCGCAGACAAACTCGCATTAATAAACCAACGTACAGAGTATATCTC
AGTTGGTTCAGCCGTACAGGCTGTAATGCGTCCATACATCTTCAGGGATATTTTATTCCCGGAGATGTCA
GCTAAGCATGGCATTTGTCCGTTCCGGACAAAAGCCTATGCGAACCTGCACTGGAAACGACGGTTCGAAC
GTTTCTATAGTGCGGTTGAGGGATTCCTGAGTAAATCTGAGGAAGCTCGAGAACAGATTAATTCTGTCTC
AGAGTCCGAGATGGATTCTCTCCATGAACGGATTTTATCCATCATGGAAGACCCGTCCCGTCCCGATATC
CACCGCGAGGTAGTTTTCCTTCCTCGCAAAGTGGTTTTTACAGAGACTTTATGTACTCTGCAATTACCCG
GGATCTATTAAAGACCCTTTACATGATCATTTACTGATCATCCGCTTTACGCGGCGGGTATATCTCTCGT
GAGGATGGATTAAATCCTGCGCGCACCTTACGACCAGGCGTACCCGGTACGGGGGTTGCTATACGCTCCT
CCGACATCTGATTAACAGATGTTTGCTGTTTTAAACAGCAAAAAAAAAAAAA

>CAK8177718.1 RNA-dependent RNA polymerase [Narnavirus sp.]
MSLPDIVPVHDCRTSAQEVRDGTIKVDHRWYLDFLDTNPGSILHGWDEIPSYARKRRVIRSFPFELKTDV
IDKLSRADSEFVKRIVNLVLAFEDNLLFSDPEHFPRIFRSREYNRLFKWVVSSEVYSPGSPVDEWKSFIQ
VVRWHAFRSQTEKPDGPRNFPGFRRQKGLDELHHFWAVLVPWLIPVWREGLTSKLQCTRLMHFLSSRNMP
AAPKKKRNASLEEHFRVLTSEGPDLASRNEWLRGAAFHYGRRLRSESGETISGAHLSVTNSASFLSSVKE
GGRGVETASFFRSWIMKKPETTEQRLTWFGKPYWLEKGTPQWETMCREVKPQSGEPGDSSYLTEVDFTPG
AFRYEDPIYCLDQYTGYQIFQWSLEEGISRGYIRGKPYRDPDGLSITGKAPLIRASAIGEPGNKSRVVTV
GEGWLTIFLQPYAHALIGILKRDSDAKAGLSRAWQGFEYVKRWAGRGKPPPEEQRFILSSDLKTATDYCP
HGYSRSLLKGFIDGLGLETDLTNLWVDLLTCGRTYVSPFGKKITTRGILMGDPGAKLVLSLFNKVAEMEA
QVRFNIGTHLRMSGKFLLRHLETDGLHGPQSRLFAFAGDDHVALGPVNYLRQITASHIRNGMCVSTTTNY
ISCVAAVYCEEMVFIRSTDVWKYWGSNIPLYKSPYQENPHVDALKLRLLSLCRKEHEGKNDANPAIGMSS
SLRGMIHWFAEGWETTRGIVSKRFRQKMRGLIPENLFLAALPKELGGIECPVFDVPDTELQRHFLELPEL
HKRVIALVLRKDEKTNPSFRRMLSQIPSMSALRGVDPDSIEESVREVLSSELCQSLRLSEIRVKLGFSEV
AWEQLRFADKLALINQRTEYISVGSAVQAVMRPYIFRDILFPEMSAKHGICPFRTKAYANLHWKRRFERF
YSAVEGFLSKSEEAREQINSVSESEMDSLHERILSIMEDPSRPDIHREVVFLPRKVVFTETLCTLQLPGI
Y
```

## Question 3 : Interrogate the Virus `(20 points)` ----------------------------

**Q.** `(10 points)` Research and identify what are at least 10 *features* and 
10 *observations* regarding your virus.

Ideally this section should capture 50+ one line observations, almost like a
"scratch pad" to document your thinking. If you rule something out, keep it and
mark the line with [X]. This is "bank" of accurate statements from which you will
interpret, and build your analysis upon

List or write short sentences describing those features here. Examples are:
```
# Example Answer
- [X] Narnaviruses have a DNA genome
- Narnaviruses have an RNA genome
- Narnaviruses have a single-stranded genome
- The typical genome length is between 2,200 - 3,200 nucleotides long
- Narnaviruses typically encode for a single protein, RdRp
- Narnavirus RdRp are typically 800-1000 amino acids long
- [X] Narnaviruses infect fungi, like yeast
- Narnaviruses infect apicomplexans, like toxoplasma
- ...
```
**A.** TODO

**Q.** `(10 points)`  Create a figure(s) visualizing your virus genome (see 
papers for examples/inspiration). The figure should capture the *key genomic*
*features* of the virus which includes (but is not limited to):
- Open reading frames, and note if they are not CDS complete
- Annotation of every ORF
- A scale legend in nucleotides and amino acids for ORFs

In addition consider virus-family specific features, such as
- RNA secondary structures
- Conserved sequence motifs
- Frameshifting sites
- Evidence of poly-A or splicing
- Additional known (or unknown) protein domains

In addition consider additional measurements, such as
- TPM / RPKM for the virus within your dataset
- Additional viral segments/viral contigs
- Coverage profile of reads mapped to your contig
- Single Nucleotide Variants present in the mapping data

The Figure Legend must contain sufficient methods description such that your
analysis is reproducible. Include supplementary data files if need be.

- `(6 points)` Figure (technical)
- `(2 points)` Figure (aesthetic)
- `(2 points)` Figure legend

**A.** TODO

**Q.** - `(bonus 2 points)`
  - Run the RdRp and/or open reading frame through AlphaFold
  - Add a figure of the RdRp with motifs A (blue), B (green), and C (red) visualized.
    See: `Odysseyus Narnavirus` example paper. Try PyMol.

**A.** TODO

## Question 4 : `(5 bonus points max)` -----------------------------------------

**Q.** `(bonus 5 points)` 
  Do something creative or cool with the viral genome. The endevour should be
  creative `3 points` and yield an insight that could not otherwise be obtained
  `2 points`.

**A.** TODO

## Question 5 : Discussion `(5 points)` ----------------------------------------

Highlight for your reader what you have learned from this virus, or what novel
biological insights does characterizing this genome offers to the reader. What
facts did you find the most fascinating about your kind of virus, and did your 
virus share these traits? Even more interesting is if it _doesn't_ have a trait
known to that virus. It's always fun to push against the textbook definitions.
(3 sentence and 50 - 150 word limit)

**A.** TODO

## Question 6 : References `(10 points)` ---------------------------------------

Add at least **10 relevant references** throughout your report. Be sure to 
reference both software papers, biology papers, and the paper from which the
data was generated (if available).

## Question 7 : `(5 bonus points max)` -----------------------------------------

```
Twitter-like platforms are powerful means to communicate key messages of a 
scientific work. Compose a Tweet-torial of 1-3 tweets (280 character limit + 
option for image) to highlight the story of your scientific report.

Example: https://x.com/RayanChikhi/status/1963157152958210378
```

**A.** TODO


## Question 8 : `(30 points)` --------------------------------------------------

The rate of scientific progress is accelerating. We are saturated in research
results, making it difficult to resolve your work from everyone else. In this 
landscape, your science communication must evolve to stand out.

**Q.** - `(30 points + 5 Bonus for creative use of the medium)`
Create a video (upto 8 minutes) to tell the story of your virus in the most 
compelling way possible.

-- OR --

**Q.** - `(30 points)`
Congratulations, your scientific abstract has been selected for a short 'Oral 
Presentation' for the `3rd RNA VIRUSxDISCVRY Symposium`! Please prepare a
**8 minute scientific talk** for the symposium. For your convenience we have
provided a _suggested_ outline for the presentations.

You probably will not have the time to present _all_ of your findings, you
can and should prioritize the most compelling 

- `(1 points)`: Title Slide
  Don't under-estimate the importance of a good first impression.
  Include a professional title slide which includes
  - Your name (prominently)
  - The date
  - Venue you are presenting
  - A research title. Make it engaging, or your main scientific conclusion
  - A visually compelling image related to your work

- `(4 points)` : Background (~2 slides)
  The primary purpose of scientific talks is to educate. You have been learning
  and exploring the subject matter, and along the way you have uncovered a new
  perspective on your subject. What background information did you find
  the most fascinating? Share that! Others will be fascinated too.
  
  (1) Introduce the host/environment in which your virus was found in,
  and describe the societal relevance of this system. Why was it being
  studied? What are the big questions in the field?
  
  (2) Introduce the kind of virus which you will be describing, what are the
  expected properties of these viruses, and which species of this type are the
  most prominent / well known?
  
- `(3 point)` : Hypothesis / Research Question
  Using the information in the background, what do you expect to find? If you
  write a hypothesis it must be falsifiable (measurable).
  
  A great hypothesis is something which is a clear logical extension of the
  background information.
  
- `(5 points)`: Methods (~2 slides)
  Describe what you did to address the Research Question in the appropriate
  amount of detail. For a talk you do not have to provide every detail of your
  method, but you must provide the logical formulation of what you did. 
  Parameters / method details can be written on the slide as part of the visuals
  while your oral discussion focuses on the concepts and rationale for this
  approach.

- `(5 points)`: Results (~2 slides)
  
  (1) See: Question 2 -- Digital Ecology
  
  (2) See: Question 3/4 -- Interrogating your Virus

- `(5 points)` : Discussion (1 slide)
  See: Question 5

- `(2 points)` : Conclusion (1 slide)
  What are the 3-4 most important conclusions which you can drive home. Bring
  home the key take-aways here.

- `(5 points)` : Effective use of slide visuals (clarity, colors, emphasis),
   logical organization of ideas, and clarity of the narrative. 

**How to prepare a scientific talk**
- [NSF Guide for Scientific Talks](https://www.cgd.ucar.edu/research/scientific-talk)
- [How to tell a compelling story in scientific presentations](https://www.nature.com/articles/d41586-021-03603-2)
- [Effective slide design](https://pedsresearch.org/uploads/blog/doc/Designing_Effective_Scientific_Presentations_from_SfN.pdf)
