# Novel discovery of a geographically distributed partiti-like virus with uncertain host range and taxonomical placement revealed through ecological and machine learning analyses


### Video Thumbnail
[Video thumbnail](img/Durnavirus_ambigiformis/thumbnail.png) 


### Video Description
presented by: [Jeremy Chan](https://github.com/jeremyschan)

project link: [https://github.com/ababaian/VIRUSxDISCVRY/virus/Durnavirus_ambigiformis.md](https://github.com/ababaian/VIRUSxDISCVRY/virus/Durnavirus_ambigiformis.md)


Learn a little about this sweet-like-honey partiti-like virus, Durnavirus ambigiformis, and about its inability to classify into existing viral taxonomy.


Virus discovery through RNA sequencing continues to uncover lineages that challenge the classical definitions of viral taxonomy. Partitiviridae comprise a family of bisegmented double stranded RNA (dsRNA) viruses known for infecting plants, fungi, or protozoa [Vainio et al., 2018]. However, there remains a large number of novel viruses that remain unclassified within this family, otherwise known as partiti-like viruses. Here, we show that a novel partiti-like virus that does not fit into any established genus within the Partitiviridae family. We found that the virus ecologically resembles members of the cryspovirus genus by appearing across diverse species and geographic locations, indicating that a broad ecological distribution is not exclusive to cryspoviruses and may occur in other unclassified partitivirus-like lineages. Furthermore, analysis of the *de novo* RdRp contig revealed that when compared to other partitiviruses and other families within the order Durnavirales, it is closest to Picobirnaviridae rather than any established partitivirus genus, revealing that the virus may not fit into Partitiviridae at all, and instead may indicate a new family. We anticipate that these result will be valuable for virus discovery by highlighting the need for more sophisticated classification strategies for the growing number of unassigned “-like” viral lineages. As viruses evolve quickly and continually generate novel sequence space, methods that can resolve these ambiguous groups will be essential for keeping known taxonomical groups aligned.

