# Reading Between the Strands: Dual-Coding Glossinamyces chrysopsiformis Supports Hidden Complexity in Narnaviruses

### Video Thumbnail

![Video thumbnail](../img/Glossinamyces%20chrysopsiformis/thumbnail.png)

### Video Description

presented by: [Rayna_Ma](https://github.com/Rayna-Ma)

project link: [Glossinamyces chrysopsiformis](https://github.com/ababaian/VIRUSxDISCVRY/virus/Glossinamyces%20chrysopsiformis.md)

Discovery of a novel dual-coding narna-like RNA virus from the tsetse fly holobiont

{ ---- Text Fold ---- }

Viruses are broadly classified into DNA and RNA viruses. RNA viruses are characterized by compact genomes and replication mediated by RNA-dependent RNA polymerase (RdRp).

Narnaviruses are among the simplest RNA viruses known, classically defined as positive-stranded genomes encoding only one single ORF for the RdRp. They infect unicellular eukaryotes such as fungi. Lacking both a capsid and extracellular virion, they represent an extreme form of genomic minimalism and have long served as models for the low limits of viral complexity.

However, recent metagenomic discoveries have begun to challenge this simplicity. In particular, emerging reports of long complementary-strand ORFs contradict the canonical view that narnaviruses encode only a single protein.

**Here we show** a newly detected narna-like virus, *Glossinamyces chrysopsiformis*, prevalent across *Glossina morsitans* (tsetse fly) associated RNA sequencing datasets.

Our results demonstrate that *G. chrysopsiformis* is a dual-coding narnavirus, containing both a canonical 760 aa RdRp and a 744 aa reverse ORF. This reinforces that complementary-strand coding is a recurring feature rather than an anomaly within this group. On the other hand, instead of infecting the tsetse insect or its parasite, the virus is most consistent with infecting a unicellular eukaryote holobiont broadly distributed across *G. morsitans* tissues.

These findings expand the known genomic architecture of Narnaviridae and highlight that dual-coding strategies may be widely deployed in insect-associated lineages.

I anticipate this work to illustrate how hidden coding capacity can reshape our interpretation of ultra-simple RNA viruses, suggesting that even the most minimal genomes may encode uncharacterized biological functions. This framework provides a foundation for future efforts to define the true genetic and ecological diversity of narnaviruses.
