# Viralzone
written by: [Howard Cheng](https://github.com/HowardCheng2005)

[15 minutes] 
ViralZone is a web based resource which provides 'fact sheets' for known viral families. It takes in virus family searches and offers general virus information along with structural models and access to associated NCBI nucleotide and UniprotKB protein entries. This allows researchers to easily bridge the connection between textbook information with genomic and proteomic sequences.


**Tutorial Objective**: Through this tutorial on ViralZone, the reader should be able to access the facts sheets, diagrams, and UniProtKB associated protein sequences for their chosen viral family.

## Input / Prerequisites
- access to [ViralZone](https://viralzone.expasy.org/)
- access to the web
- A virus family (use partitiviridae for example)

## Output

A webpage containing the general 'fact sheet' of the viral family that includes virion and genomic structure, replication process and taxanomic information along with associated protein sequences and 3d structures via the UniProtKB database links. Output can be utilized to find associations within the virus.  

### 1. Navigate to [ViralZone](https://viralzone.expasy.org/)

### 2. Find viral family through search bar
By putting in your viral family/genera in the search bar, you are able to find any ViralZone entries associated with it. From this page, click on the link that best matches with your chosen viral family. For this example, click on the Partitiviridae page.

![](photos/image1.png)

### 3. (Optional) Finding viral genus/family through taxonomic and host searches
Another way of finding your viral family is by either searching up its taxonomic grouping or its host. In this tutorial, we will cover both methods.

#### 3.1 searching via taxonomic grouping
Go to the main page, and under “Browse Factsheets” click the “By Taxonomy" option to enter the taxonomic grouping page.

![](photos/image2.png)

In this page, the virus groups are split via the Baltimore classification system. For our example, we know that the partitiviridae is part of the Pisuviricota Phylum, so we enter that section. There, we will then be able to locate our Partitiviridae family under the Duplopiviricetes class

![](photos/image3.png)

#### 3.2 Searching via host

On the main page under “Browse Factsheets”, select the “by Host” option to take you to the host classification page. 

![](photos/image4.png)

Here, the virus groups are separated by its potential hosts and sorted under the Baltimore classification system. The potential hosts that a viral family can infect is shown by the color(s) beside the entry links. In our example, to get to the Paritiviridae section, we can click on either the plant or fungi group and look under the ssRNA(-) section to locate our page.

![](photos/image5.png)

### 4. General information page.
Here, we enter the general ‘fact sheet’ of the viral family. In this page, we can find the general information about the virus, with information about the virion and genomic structure, the gene expression, related enzymes, taxonomy, replication steps, database links and more. 

Within this page, there is a list of related viral protein sequences under the “Virus” dropdown, and an assortment of reference strains that were manually chosen for standardization and alignment. To access more information, click on the sequence, genome assembly, and proteome links associated with each reference strain for their respective NCBI/UniProt pages. 

![](photos/image6.png)

### 5. Accessing uniprot pages
To specifically find the protein sequence and functional annotations of your associated viral strain, you can click onto the “Proteins by Strain” or “Proteins by Name” sections in order to access the virus’s associated UniProtKB/Swiss-Prot entries. Depending on the section, these entries are either grouped by their reference strain or Protein name.

To access the databases, click on the link associated with the reference strain/ protein name to access its Uniprot page, which will provide structural and functional information about the virus sample. In this tutorial, look at the UniProtKB entry for RDRP_AHV2H

![](photos/image7.png)

![](photos/image8.png)

### 6. 3D structure of associated virus sequences
To get the 3D folding of each associated viral protein sequence, click on the “3D Models” section of the page to get the Uniprot entries for each associated virus. These links will include a 3D structure of the protein, which was generated by Alphafold.  For this tutorial, click on any viral sequence and look at its 3D folding structure on UniProt.

![](photos/image9.png)

### 7.Host virus interaction:
If the virus family/genus has any known host-virus interactions, the page will include a section called “Host-virus interaction” to click on. This will include all known potential functions of the viral family, and provide the relevant links to the sources. In our tutorial, our viral family does not have any known interactions, so please click on another sample with a known Host-virus interaction for further exploration.

![](photos/image10.png)

### Conclusion

That's it! You've used the ViralZone website to find the structural and functional entries for your viral family!

From this tutorial, we have provided an overview on how to apply the ViralZone website in order to help researchers find general information and database entries about their viral family/genus. Through this, they should be able to use this site in order to find associated protein sequences along with their functional annotations within the NCBI/Uniprot databases. This tool then acts as a great starting point in virome discovery, as it provides generalized information about your chosen virus aligned with some associated database links for further research.

### See Also:

- [ViralZone: a knowledge resource to understand virus diversity](https://academic.oup.com/nar/article/39/suppl_1/D576/2508477?login=false#93000909)